docs/governance/project-lifecycle.md

```diff
@@ -99,13 +100,14 @@ End users should evaluate Labs with care, as this stage does not set requirement
 
 To be considered for the Labs Stage, the project must meet the following requirements:
 
-* Complete an [project intake form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). This intake form will create:
-    * A project charter ([sample](https://docs.google.com/document/d/1Sxnktu3d_xLFSZmAmSFbUp1Y1s1E36YvNDupY2AmigU/edit)) that documents an intellectual property policy that leverages the Apache 2.0 license or a permissive open source license.
-    * A collaboration agreement that in the case of existing projects, provides an agreement to transfer the project name, trademarks, and electronic account assets (github repo, social media accounts, domain names, etc.) to Linux Foundation Europe for the benefit of the OpenWallet Foundation.
 * Submit a [project proposal](https://github.com/openwallet-foundation/project-proposals#instructions-for-proposing-a-project).
-
+* If the Lab is accepted by the TAC, the OWF Lab Project will then complete [project intake form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). This intake form will create:
+  * A project charter [(sample)](https://docs.google.com/document/d/1Sxnktu3d_xLFSZmAmSFbUp1Y1s1E36YvNDupY2AmigU/edit) that documents an intellectual property policy that leverages the Apache 2.0 license or a permissive open source license.
+  * A collaboration agreement that in the case of existing projects, provides an agreement to transfer the project name, trademarks, and electronic account assets (github repo, social media accounts, domain names, etc.) to Linux Foundation Europe for the benefit of the OpenWallet Foundation.
+    
```

Comment:
```suggestion
```

I don't think we need this since you added this at line 47.

Reply:
The 3rd element about the collaboration agreement is not part of what we have in line `47`. Should we keep it here?

Reply:
We should probably move it to line 47 as it is relevant for all project stages.

Reply:
I have made this change.